README.md

```diff
@@ -13,10 +13,15 @@ removes the need for a privileged, `NET_ADMIN` container in the Istio users' app
 
 ## Usage
 
-The [Istio Helm charts](https://github.com/istio/istio/tree/master/install/kubernetes/helm/istio) integrate
-the option to install the Istio CNI.  The [Istio Installation with Helm](https://preliminary.istio.io/docs/setup/kubernetes/helm-install/)
-procedure with the addition of the setting `--set istio_cni.enabled=true` enables the Istio CNI for
-the Istio installation.
+A complete set of instructions on how to use and install the Istio CNI is available on the Istio documentation sites under the "Istio Install Istio with the Istio CNI plugin".  Only a summary is provided here.  The steps are:
+
+1. Install Kubernetes and kubelet in a manner that can support the CNI
+
+2. Install Kubernetes with the [ServiceAccount admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#serviceaccount) enabled
+
+3. Install the Istio CNI components
```

Comment:
For example, the following example uses the local version of the CNI repo's helm chart to install specific Istio CNI plugin containers (see the `hub` and `tag` helm parameters):

```
$ CNI_HUB=docker.io/my_userid
$ CNI_TAG=mytag
# run from the ${GOPATH}/src/istio.io/cni dir (repo where istio/cni was cloned)
$ helm template --name=istio-cni --namespace=kube-system --set "excludeNamespaces={}" --set hub=${CNI_HUB} --set tag=${CNI_TAG} --set pullPolicy=IfNotPresent --set logLevel=debug  deployments/kubernetes/install/helm/istio-cni > istio-cni_install.yaml
```

Reply:
Ok can add a specific example